<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfomarksEditDlg</class>
 <widget class="QDialog" name="InfomarksEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>425</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>425</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>395</width>
    <height>510</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Edit Markers</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="2" column="0">
    <widget class="QGroupBox" name="selectedMarkBox">
     <property name="title">
      <string>Fields:</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="objectType">
        <item>
         <property name="text">
          <string>Text</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Line</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Arrow</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel1_2">
        <property name="text">
         <string>T&amp;ype:</string>
        </property>
        <property name="buddy">
         <cstring>objectType</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="textLabel3_1">
        <property name="text">
         <string>C&amp;lass:</string>
        </property>
        <property name="buddy">
         <cstring>objectClassesList</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QComboBox" name="objectClassesList">
        <item>
         <property name="text">
          <string>Generic marker</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Herb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>River</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Place</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mobile name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Road</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Object</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Locality</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QGroupBox" name="positionGroupBox">
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>115</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>340</width>
          <height>115</height>
         </size>
        </property>
        <property name="title">
         <string>Position:</string>
        </property>
        <layout class="QGridLayout">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel4_3">
           <property name="text">
            <string>X1</string>
           </property>
           <property name="buddy">
            <cstring>m_x1</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="textLabel4_3_3">
           <property name="text">
            <string>X2</string>
           </property>
           <property name="buddy">
            <cstring>m_x2</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="textLabel4_3_4">
           <property name="text">
            <string>Layer:</string>
           </property>
           <property name="buddy">
            <cstring>m_layer</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QSpinBox" name="m_layer">
           <property name="minimum">
            <number>-10</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel4_3_2">
           <property name="text">
            <string>Y1</string>
           </property>
           <property name="buddy">
            <cstring>m_y1</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="textLabel4_3_2_2">
           <property name="text">
            <string>Y2</string>
           </property>
           <property name="buddy">
            <cstring>m_y2</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="textLabel4_3_5">
           <property name="text">
            <string>Angle:</string>
           </property>
           <property name="buddy">
            <cstring>m_rotationAngle</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="m_x2">
           <property name="minimum">
            <number>-2147483647</number>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="m_y2">
           <property name="minimum">
            <number>-2147483647</number>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="m_y1">
           <property name="minimum">
            <number>-2147483647</number>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="m_x1">
           <property name="minimum">
            <number>-2147483647</number>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QDoubleSpinBox" name="m_rotationAngle">
           <property name="suffix">
            <string>Â°</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>45.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="startLabel">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLabel" name="endLabel">
           <property name="text">
            <string>End</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="objectText"/>
      </item>
      <item row="10" column="0" colspan="3">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="objectModify">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="objectCreate">
           <property name="text">
            <string>&amp;Create</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="textLabel2_2">
        <property name="text">
         <string>&amp;Text:</string>
        </property>
        <property name="buddy">
         <cstring>objectText</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="objectsList"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1_2_2">
     <property name="text">
      <string>&amp;Marker:</string>
     </property>
     <property name="buddy">
      <cstring>objectsList</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>objectsList</tabstop>
  <tabstop>objectType</tabstop>
  <tabstop>objectText</tabstop>
  <tabstop>objectClassesList</tabstop>
  <tabstop>m_x1</tabstop>
  <tabstop>m_x2</tabstop>
  <tabstop>m_layer</tabstop>
  <tabstop>m_y1</tabstop>
  <tabstop>m_y2</tabstop>
  <tabstop>m_rotationAngle</tabstop>
  <tabstop>objectModify</tabstop>
  <tabstop>objectCreate</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
